pkgs/build-support/autoconf-cache/src/configure.ac

```diff
@@ -0,0 +1,136 @@
+dnl This is check-all script, based on one found in GNU Hello distribution.
+dnl Idea is that check as much as possible during build of this derivation
+dnl and reuse
+
+dnl Copyright (C) 2004-2019 Free Software Foundation, Inc.
+
+dnl This file is free software; as a special exception the author gives
+dnl unlimited permission to copy and/or distribute it, with or without
+dnl modifications, as long as this notice is preserved.
+dnl
+dnl This program is distributed in the hope that it will be useful, but
+dnl WITHOUT ANY WARRANTY, to the extent permitted by law; without even the
+dnl implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+AC_INIT([Autoconf Cache], [current])
+
+dnl Must come before AM_INIT_AUTOMAKE.
+AC_CONFIG_AUX_DIR([build-aux])
+AM_INIT_AUTOMAKE([1.11.1 readme-alpha subdir-objects foreign])
+AC_PREREQ([2.64])
```

Comment:
Do these version numbers need to match the ones in nixpkgs?

Reply:
These are lower bounds.

Reply:
Maybe I can skip some of them, I'll look.

Reply:
AC_PREREQ is actually optional